nautobot/ipam/filters.py

```diff
@@ -316,10 +316,19 @@ def search_contains(self, queryset, name, value):
         except (AddrFormatError, ValueError):
             return queryset.none()
 
+    def generate_query_filter_present_in_vrf(self, value):
+        if isinstance(value, str):
+            value = VRF.objects.get(pk=value)
+
+        # This stringification is done to make the `pretty_print_query()` output look human-readable,
+        # and nothing more. It would work as complex objects but looks bad in the web UI.
```

Comment:
Is this still an issue after #2197?

Reply:
Yes:

<img width="1186" alt="image" src="https://user-images.githubusercontent.com/138052/184446245-68b31802-36b6-4e77-a5cf-bde7611c2b28.png">

Compared to your fixes with leaving the stringification in there:

<img width="934" alt="image" src="https://user-images.githubusercontent.com/138052/184446340-ce7f45ef-42fa-45a7-85dc-c72e95849a11.png">

Reply:
Cool, thanks. Not blocking but wonder if we can/should fix this in `pretty_print_query()` itself?

Reply:
Yeah I think that is probably better so it's not up to plugin authors or us in the future.